docs/src/pages/demos/chips/ChipsPlayground.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { Fragment } from 'react';
+import { withStyles, WithStyles, Theme } from '@material-ui/core/styles';
+import MarkdownElement from 'docs/src/modules/components/MarkdownElement';
+import Grid from '@material-ui/core/Grid';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Radio from '@material-ui/core/Radio';
+import Paper from '@material-ui/core/Paper';
+import Avatar from '@material-ui/core/Avatar';
+import Chip from '@material-ui/core/Chip';
+import FaceIcon from '@material-ui/icons/Face';
+import DoneIcon from '@material-ui/icons/Done';
+
+type ColorType = "default" | "inherit" | "primary" | "secondary" | undefined;
+
+type VariantType = "default" | "outlined" | undefined;
+
+type ActionType = "avatar" | "color" | "icon" | "onDelete" | "variant";
+
+interface ChipsPlaygroundState {
+  color: ColorType;
+  onDelete: string;
+  avatar: string;
+  icon: string;
+  variant: VariantType;
+}
+
+interface ChipsPlaygroundProp extends WithStyles<typeof styles> {}
+
+const styles = (theme: Theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  control: {
+    padding: theme.spacing(2),
+  },
+  chipWrapper: {
+    marginBottom: theme.spacing(5),
+  },
+});
+
+class ChipsPlayground extends React.Component<ChipsPlaygroundProp, ChipsPlaygroundState> {
+  state: ChipsPlaygroundState = {
+    color: 'default',
+    onDelete: 'none',
+    avatar: 'none',
+    icon: 'none',
+    variant: 'default',
+  };
+
+  handleChange = (key: ActionType) => (event: React.ChangeEvent<{}>, value: string) => {
+    this.setState({
+      [key]: value,
+    } as Pick<ChipsPlaygroundState, keyof ChipsPlaygroundState>);
+  };
+
+  handleDeleteExample = () => {
+    alert('You clicked the delete icon.'); // eslint-disable-line no-alert
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { color, onDelete, avatar, icon, variant } = this.state;
+
+    const colorToCode = color !== 'default' ? `color="${color}" ` : '';
+    const variantToCode = variant !== 'default' ? `variant="${variant}" ` : '';
+
+    let onDeleteToCode;
+    switch (onDelete) {
+      case 'none':
+        onDeleteToCode = '';
+        break;
+      case 'custom':
+        onDeleteToCode = 'deleteIcon={<DoneIcon />} onDelete={handleDelete} ';
+        break;
+      default:
+        onDeleteToCode = 'onDelete={handleDelete} ';
+        break;
+    }
+
+    let iconToCode;
+    let iconToPlayground = <Fragment />;
```

Comment:
Can it accept `undefined`?

Reply:
Then... I don't now because I did not found a way to test, because the option to change for TS is not being displayed. Plis can you help me to resolve this problem, maybe exists a conflict with import MarkdownElement from 'docs/src/modules/components/MarkdownElement